Makefile

```diff
@@ -141,6 +146,12 @@ run_debian: salus
 		-device e1000e,netdev=usernet \
 		$(EXTRA_QEMU_ARGS)
 
+run_salus_tsm: salus_tsm
+	$(QEMU_BIN) \
+		$(MACH_ARGS) \
+		-kernel $(RELEASE_BINS)salus-tsm \
+		$(EXTRA_QEMU_ARGS)
```

Comment:
We should probably have an internal discussion about how this Makefile target is expected to be used.